### DISCUSSION TITLE
Clarification on Mono's Support for ConcurrencyMode.Multiple in WCF Services

### DISCUSSION THREAD

**Discussion by yashnikam** [mono/mono#21817] [Q&A]
Hello,

I am working on hosting a WCF service using Mono with ServiceHost and ServiceBehavior. I want to configure the service to handle multiple concurrent requests by setting ConcurrencyMode to Multiple.

Could you please confirm whether Mono fully supports ConcurrencyMode.Multiple and if there are any known limitations? Additionally, are there specific Mono versions or configurations required for this to work as expected?

Thank you for your assistance!

Best regards,
Yash N

**Comment by lextm** [comment]
There wasn't major progress on Mono WCF ever since Moonlight, so this page contains all you might be able to find https://www.mono-project.com/docs/web/wcf/

CoreWCF on .NET 8 is more reasonable a migration path for you.
